src/PoshGitTypes.ps1

```diff
@@ -101,6 +131,32 @@ class PoshGitTextSpan {
         $this.CustomAnsi = $null
     }
 
+    [string] ToEscapedString() {
```

Comment:
Isn't this whole method equivalent to `EscapeAnsiString $this.RenderAnsi()`?

Reply:
Yup.  Done.